CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 1. [15306](https://github.com/influxdata/influxdb/pull/15306): Added missing string values for CacheStatus type
 1. [15348](https://github.com/influxdata/influxdb/pull/15348): Disable saving for threshold check if no threshold selected
 1. [15354](https://github.com/influxdata/influxdb/pull/15354): Query variable selector shows variable keys, not values
+1. [15246](https://github.com/influxdata/influxdb/pull/15427): UI/Telegraf filter functionality shows results based on input name
```

Comment:
@desa this isn't auto generated?

Reply:
It should be, but at the moment it isn't
Might be worthwhile to schedule some time to get it auto-created from we have this tool which the flux team uses https://github.com/influxdata/changelog, but no one has set it up on influxdb yet